Lib/glyphsLib/classes.py

```diff
@@ -1931,6 +1960,45 @@ def __repr__(self):
         else:
             return "<GSHint %s %s>" % (self.type, direction)
 
+    @property
+    def originNode(self):
+        if self._originNode is not None:
+            return self._originNode
+        if self._origin is not None:
+            # Find the GSNode that is refered to by the indices in _origin
+            path_index, node_index = self._origin
+            layer = self.parent  # FIXME: (jany) I don't have access to the parent from the GSHint!!!
+            path = layer.paths[path_index]
+            node = path.nodes[node_index]
+            return node
```

Comment:
@schriftgestalt I have a question about this piece of code: when I read `origin` from the file, I have a path id & node id. In order to return the matching `GSNode`, what I want to do is: from the GSHint where I am, I want to go up one level to find the layer, and access the path and node by id. Except I don't have `GSHint::parent`, according to the Python API documentation, so I can't go up. Should I add the parent to this class? Or is there another way? Or maybe it's not the right place to do that? Thanks in advance